Cargo.toml

```diff
@@ -1,10 +1,22 @@
 [workspace]
 members = [
-	"node",
-	"pallets/*",
-	"primitives",
-	"runtimes/*",
-	"support",
+  "support",
+  "pallets/emergency-shutdown",
+  "pallets/allocations",
+  "pallets/amendments",
+  "pallets/grants",
+  "pallets/poa",
+  "pallets/reserve",
+  "pallets/tcr",
+  "pallets/root-of-trust",
+  "pallets/root-of-trust/rpc",
+  "pallets/root-of-trust/rpc/runtime-api",
+  "pallets/staking",
+  "pallets/mandate",
+  "primitives",
+  "runtimes/main",
+  "runtimes/staking",
+  "node",
```

Comment:
I'd like to keep the initial settings here so that we do not have to update that file manually every time

Reply:
Sure Eliott, will restore it back :+1: